noaa_semantic_search/operations/hybrid_search.json

```diff
@@ -52,6 +52,47 @@
         }
       }
     },
+    {
+      "name": "bool-only-term-range-date-sort",
```

Comment:
why we need this operation as part of the workload? same is for all others with bool query.
If you want to have some baseline you can add bool with sorting for only medium subset. That should be enough to see if some deviation is there. Others will just increase running time without adding a value